bulk_sync_into_benchling/README.md

```diff
@@ -0,0 +1,54 @@
+This script demonstrates how to use the bulk create and register entity endpoints. The script uses long running tasks and demonstrates how to handle them. In this example, we are importing antibodies defined in a JSON file:
+
+- Each Antibody entity links to a Heavy Chain and a Light Chain entity
+- The Heavy Chain and Light Chain are amino acid (AA) sequences
+
+The antibodies to import are provided in the following format:
+
+```
+{
+  "name": "AB-BRCA2-0001",
+  "Heavy Chain": "AAAAAAAAAAAAAAAAAAAAAAAA",
+  "Light Chain": "CCCCCCCCCCCCCCCCCCCCCCCC"
+}
+```
+
+# Prerequisites
+
+This script requires the Registry and Molecular Biology applications. In addition, the script expects two **entity schemas** to be configured ([instructions](https://help.benchling.com/articles/2725066-configure-your-registry)).
+
+- An Antibody schema with Heavy Chain and Light Chain fields: ![Antibody fields](images/antibody-fields.png)
```

Comment:
Is this a custom-entity schema?

Reply:
I think its an aa-sequence with a schema.